conf/modules.config

```diff
@@ -776,6 +776,20 @@ process {
         ]
     }
 
+    withName: MOTUS_PREPLONG {
+        tag = {"${meta.db_name}|${meta.id}"}
+        ext.args = {[
+                "-sl ${params.motus_longprep_splittinglength}",
+                "-ml ${params.motus_longprep_minlen}"
+            ].join(' ').trim() }
+        ext.prefix = params.perform_runmerging ? { "${meta.id}_${meta.db_name}.prep_long" } : { "${meta.id}_${meta.run_accession}_${meta.db_name}.prep_long" }
+        publishDir = [
+            path: { "${params.outdir}/motus/${meta.db_name}/" },
+            mode: params.publish_dir_mode,
+            enabled: params.motus_save_long_reads
```

Comment:
Maybe should be 'save cutup' long reads? It's not the actual long reads that are being saved right?

Reply:
Hmm, true. Maybe `save_split_long_reads`?